internal/schema/type.go

```diff
@@ -82,3 +82,21 @@ func (t *Type) GetTags() string {
 
 	return jsonTag + "\"`"
 }
+
+// GoType is used to determine if a type in NerdGraph is already a native type of Golang.
+func (t *Type) GoType() bool {
+	goTypes := []string{
+		"int",
+		"string",
```

Comment:
`float64` is at least due here.  What else?

Reply:
This is what I've run into thus far.  I'll avoid thinking too far ahead here, since floats might be handles special, given the nil case.